test_build_tag_push.py

```diff
@@ -0,0 +1,99 @@
+import unittest
+import build_tag_push
+import os
+import subprocess
+import argparse
+from unittest.mock import patch
+import io
+import sys
+
+path_to_test_yml = ('/Users/alyssakelley/Desktop/MVPStudio/build_tag_push/'
```

Comment:
we need this to be able to work on anybody's machine. I think we talked about how to do the `os.path.abspath(os.path.dirname(__FILE__))` trick to get the path to this file and then, from that, compute the path to your test files. That way anyone can check out our open-source repo and run the tests.